source/SnapDump-Client/SDHTTPStore.class.st

```diff
@@ -98,12 +112,12 @@ SDHTTPStore >> snapshotsWithVersion: aVersion [
 SDHTTPStore >> versionsOfProject: aProject [ 
 	| response |
 	response := self client
-		url: uri / #projects / aProject name / #versions;
+		url: uri / #projects / aProject name urlEncoded / #versions;
```

Comment:
Why is there a urlForSnapshot: but no #urlForProject: , #urlForVersion: they can use each other and so there is only one place for the urlEncoded

Reply:
Right, I guess I have missed that !